bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -70,7 +70,7 @@ spec:
 
           exec ovn-northd \
             --no-chdir "-vconsole:${OVN_LOG_LEVEL}" -vfile:off \
-            --pidfile=/var/run/openvswitch/ovn-northd.pid \
+            --pidfile=ovn-northd.pid \
```

Comment:
According to the man pages (at least for 2.11) this will make it put the pidfile in `.`

However, do we actually need to be writing the pid files at all? It looks like the only pid file we ever read is the ovn-nbctl daemon's, and the others are "write-only" and we could just remove the `--pidfile` arguments...

Reply:
Changed. Dropped unneeded pid files.